examples/CMakeLists.txt

```diff
@@ -48,3 +48,15 @@ IF(DEAL_II_COMPONENT_EXAMPLES)
 
   MESSAGE(STATUS "Setting up examples - Done")
 ENDIF()
+
+#
+# Add a dummy target to make files known to IDEs like qtcreator
+#
+
+FILE(GLOB _misc
+  ${CMAKE_CURRENT_SOURCE_DIR}/step-*/*.cc
+  ${CMAKE_CURRENT_SOURCE_DIR}/step-*/*.h
+)
+
+ADD_LIBRARY(example_misc_files EXCLUDE_FROM_ALL ${_misc})
+SET_TARGET_PROPERTIES(example_misc_files PROPERTIES LINKER_LANGUAGE C)
```

Comment:
This has the advantage that the ```examples_misc_files``` target is actually meaningful. You can simply build it.

The problem with your ```ADD_LIBRARY``` statement is that some stupid IDEs will try to compile it (if I remember correctly).